tams_ur5_setup_bringup/launch/kinect2.launch

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<launch>
+	<include file="$(find kinect2_bridge)/launch/kinect2_bridge.launch">
+		<arg name="publish_tf" value="true"/>
+	</include>
+	<node pkg="apriltags_ros" type="apriltag_detector_node" name="apriltag_detector">
+		<rosparam param="tag_descriptions">[{id: 0, size: 0.162}]</rosparam>
+		<remap from="/image_rect" to="/kinect2/hd/image_color_rect"/>
```

Comment:
Are you *sure* we want to use the hd topic here?
I assume CPU load is quite high when the apriltags package has to process full-hd images...